authentication/rancher/authentication.go

```diff
@@ -150,3 +156,36 @@ func (p *Provider) authDisabled() bool {
 
 	return setting.Value == "false"
 }
+
+func (p *Provider) isAdmin(token string) bool {
+	req, err := http.NewRequest("GET", p.url+"/accounts", nil)
```

Comment:
@joshwget any particular reason for not returning an error to a caller stack? here and on all lines below where error just gets translated to false